### DISCUSSION TITLE
FillField and SeeElement is not working in pop-up when user came back from any browser

### DISCUSSION THREAD

**Discussion by s-sugumar** [codeceptjs/CodeceptJS#3864] [Q&A]
browser 1 login
browser 2 login in another user
browser 1 fill the user name in popup

it will get fail when user came back from any browser fillField and seeElement is not working in pop-up

**Comment by adudyak** [comment]
browser or tabs? try switching back:
https://codecept.io/helpers/Playwright/#switchtoprevioustab

**Reply by s-sugumar** [reply]
browser
